src/modules/avformat/factory.c

```diff
@@ -425,6 +431,7 @@ MLT_REPOSITORY
 	MLT_REGISTER( producer_type, "avformat-novalidate", create_service );
 	MLT_REGISTER_METADATA( consumer_type, "avformat", avformat_metadata, NULL );
 	MLT_REGISTER_METADATA( producer_type, "avformat", avformat_metadata, NULL );
+	MLT_REGISTER_METADATA( producer_type, "avformat-novalidate", avformat_metadata, NULL );
```

Comment:
I was intentionally hiding service metadata for "avformat-novalidate" since it merely shadows "avformat" and needs special care to use. IOW, you cannot simply echo the avformat metadata. It could have its own abbreviated metadata that explains purpose and requirements and then says to see avformat. Do you need it for some reason to make this work?

Reply:
A chain loads the metadata for encapsulated producer so that it can "reflect" the properties on itself. 
https://github.com/mltframework/mlt/blob/cb4a648fd4c688ee328cc53b61e7ca2d68cd22f4/src/framework/mlt_chain.c#L129
This avoids the need for the application to be aware that it is operating on a chain instead of directly on a producer.

If avformat-novalidate is the only exception, I could check for service "avformat-novalidate" and then load "avformat" in that specific case. Let me know if you prefer that.

Reply:
"avformat-novalidate" is currently the only producer like this, but I could see this coming to qimage and pixbuf. How about handling anything ending with "-novalidate"?

Reply:
done